paper/paper.bib

```diff
@@ -0,0 +1,127 @@
+  year = {2025},
+  month = {Aug},
+  url = {https://www.nexusformat.org},
+  note = {Accessed: 2025-08-01}
+}
+
+@techreport{YAML11,
+  author = {Ben-Kiki, Oren and Evans, Clark and Ingerson, Brian},
+  institution = {YAML.org},
+  year = 2005,
+  month = {1},
+  title = {{YAML} Ain't Markup Language ({YAML}) (tm) Version 1.1},
+  url = {https://yaml.org/spec/1.1/}
+}
+@misc{xml10,
+  author       = {Tim Bray and Jean Paoli and C. M. Sperberg-McQueen and Eve Maler and François Yergeau},
+  title        = {Extensible Markup Language (XML) 1.0 (Fifth Edition)},
+  howpublished = {\url{https://www.w3.org/TR/2008/REC-xml-20081126/}},
+  year         = {2008},
+  note         = {W3C Recommendation, World Wide Web Consortium (W3C)},
+}
+@misc{pip2025,
+  author       = {{The Python Packaging Authority}},
+  title        = {pip: The Python Package Installer},
+  year         = {2025},
+  howpublished = {\url{https://pip.pypa.io/}},
+  note         = {Accessed: 2025-07-15}
+}
+@article{Klosowski1997,
+title = {NeXus: A common format for the exchange of neutron and synchroton data},
+journal = {Physica B: Condensed Matter},
+volume = {241-243},
+pages = {151-153},
+year = {1997},
+note = {Proceedings of the International Conference on Neutron Scattering},
+issn = {0921-4526},
+doi = {https://doi.org/10.1016/S0921-4526(97)00865-X},
+url = {https://www.sciencedirect.com/science/article/pii/S092145269700865X},
+author = {Klosowski, Przemek and K{\"{o}}nnecke, Mark and Tischler, Jonathan Z. and Osborn, Ray},
+}
+@article{Könnecke2006,
+title = {The state of the NeXus data format},
+journal = {Physica B: Condensed Matter},
+volume = {385-386},
+pages = {1343-1345},
+year = {2006},
+issn = {0921-4526},
+doi = {https://doi.org/10.1016/j.physb.2006.06.106},
+url = {https://www.sciencedirect.com/science/article/pii/S0921452606013779},
+author = {K{\"{o}}nnecke, Mark},
+}
+@misc{nyamlIntegration:2024,
+  author = {{Dobener et al.}},
+  title = {GitHub PR: Adds Makefile Refinements and nyaml as Dependency},
+  year = {2024},
+  month = {Jan},
+  url = {https://github.com/nexusformat/definitions/pull/1336}
+  }
+@misc{nexusDef:2024,
+  author = {{NIAC and NeXus Definition Developer}},
+  title = {NeXus Definition GitHub Repository},
+  year = {2024},
+  month = {Feb},
+  url = {https://github.com/nexusformat/definition},
+  release = {2024.02}
+}
+@misc{w3c:xmlschema1,
+  author       = {Henry S. Thompson and David Beech and Murray Maloney and Noah Mendelsohn},
+  title        = {{XML Schema Part 1: Structures Second Edition}},
+  howpublished = {\url{https://www.w3.org/TR/xmlschema-1/}},
+  year         = {2004},
+  month        = {October},
+  note         = {W3C Recommendation},
+  institution  = {World Wide Web Consortium (W3C)}
+}
+@article{NOMAD2023,
+doi = {10.21105/joss.05388},
+url = {https://doi.org/10.21105/joss.05388},
+year = {2023},
+publisher = {The Open Journal},
+volume = {8},
+number = {90},
+pages = {5388},
+author = {{Scheidgen et al.}},
```

Comment:
Scheidgen, M. et al., better also cite here all authors, submission process will tell us if we need to abbreviate

Reply:
Addressed!